src/modules/companies/controller.js

```diff
@@ -44,4 +50,39 @@ const getReturns = async (request, h) => {
   return returns.map(rowMapper);
 };
 
+/**
+ * Gets a company for the specified ID
+ */
+const getCompany = async (request, h) => {
+  const { companyId } = request.params;
+  const company = await crmConnectors.companies.getCompany(companyId);
```

Comment:
Are the controller methods here returning data in the service model shape, or directly from the CRM?  (It should be the service models)